macros/models/_sector/transfers/transfers_native.sql

```diff
@@ -0,0 +1,182 @@
+{% macro transfers_native(blockchain, traces, transactions, native_token_address, genesis_balances=null, native_erc_transfers=null) %}
```

Comment:
When you `native_token_address` here you are referencing the wrapped version of the currency right? So for example for Ethereum you'd use the address of WETH right?

Reply:
prices has native tokens (where blockchain = NULL) which may be simpler for this case.